src/routes/devices/getAll.ts

```diff
@@ -32,17 +28,16 @@ export async function getAll (req, res): Promise<void> {
 
     for (const m of metadata) {
       list.push({
-        amtuser: amtCredentials ? amtCredentials[m.guid]?.amtuser : null,
-        conn: req.mpsService.mpsComputerList[m.guid] == null ? 0 : 1,
-        host: m.guid,
-        mpsuser: amtCredentials ? amtCredentials[m.guid]?.mpsuser : null,
-        name: amtCredentials ? amtCredentials[m.guid]?.name : null,
+        connectionStatus: req.mpsService.mpsComputerList[m.guid] == null ? 0 : 1,
+        hostname: m.hostname,
+        guid: m.guid,
         metadata: m
       })
     }
 
     if (req.query.status != null) {
-      list = list.filter(x => x.conn === req.query.status)
+      // eslint-disable-next-line eqeqeq
```

Comment:
status should be in the validator, and as an number, and this should be === without disabling the rule.